basic_flutter/lib/presentation/screens/edit_profile_page.dart

```diff
@@ -0,0 +1,179 @@
+          SizedBox(
+              width: double.infinity,
+              height: MediaQuery.of(context).size.height,
+              child: ListView(
+                children: [
+                  SizedBox(
+                      width: double.infinity,
+                      height: MediaQuery.of(context).size.height * 0.28,
+                      child: ProfileWidget(
+                        imagePath: employee.imagePath,
+                        isEdit: true,
+                        onClicked: () {},
+                        sentWidget: Positioned(
+                          bottom: 0,
+                          right: 4,
+                          child: buildEditIcon(Colors.blue.shade900, context),
+                        ),
+                      )),
+                  const SizedBox(height: 40),
+                  TextFieldWidget(
+                    label: 'Full Name',
+                    text: employee.name,
+                    onChanged: (name) => employee = employee.copy(name: name),
+                  ),
+                  const SizedBox(height: 20),
+                  TextFieldWidget(
+                    label: 'Profession',
+                    text: employee.profession,
+                    onChanged: (profession) =>
+                        employee = employee.copy(profession: profession),
+                  ),
+                  const SizedBox(height: 20),
+                  TextFieldWidget(
+                    label: 'Country',
+                    text: employee.country,
+                    onChanged: (country) =>
+                        employee = employee.copy(country: country),
+                  ),
+                  const SizedBox(height: 20),
+                  TextFieldWidget(
+                    label: 'Phone number',
+                    text: employee.phone,
+                    onChanged: (phone) =>
+                        employee = employee.copy(phone: phone),
+                  ),
+                  const SizedBox(height: 20),
+                  TextFieldWidget(
+                    label: 'Email address',
+                    text: employee.email,
+                    onChanged: (email) =>
+                        employee = employee.copy(email: email),
+                  ),
+                  const SizedBox(height: 20),
+                  TextFieldWidget(
+                    label: 'Location',
+                    text: employee.location,
+                    onChanged: (location) =>
+                        employee = employee.copy(location: location),
+                  ),
+                  const SizedBox(height: 40),
+                  ButtonWidget(
+                      text: 'Save',
+                      onClicked: () {
+                        EmployeePreferences.setEmployee(employee);
+                        Navigator.of(context).pop();
+                      }),
+                  const SizedBox(height: 150),
+                ],
+              )),
+        ]),
+      )));
+}
```

Comment:
add some commas here between the brackets so that visual studio code would auto-format them